bootstrap/src/test/kotlin/jp/co/soramitsu/bootstrap/IrohaTest.kt

```diff
@@ -214,7 +214,9 @@ class IrohaTest {
             createAccountDto("vacuumer", "notary"),
             createAccountDto("gen_btc_pk_trigger", "notary"),
             createAccountDto("admin", "notary"),
-            createAccountDto("sora", "sora")
```

Comment:
please add some assertions for added accounts in to test

Reply:
there is general assertion `accounts.forEach {assert..}` or could you be more specific on the comment?

Reply:
Ok, it's enought